laravel/resources/views/modals/administration/modal-edit.blade.php

```diff
@@ -12,21 +12,27 @@
                     <div class="col-lg-12">
                         <div class="mb-3">
                             <label for="validationName" class="form-label fw-bold">Nickname</label>
-                            <input type="text" class="form-control" id="validationName" name="name" value="{{ old('name', $userEdit->name) }}" placeholder="e.g. MyNickname" required>
-                            <div class="form-text">The users display name.</div>
+                            <input type="text" class="form-control" id="validationName"
+                                   name="name" value="{{ old('name', $userEdit->name) }}"
+                                   aria-describedby="validationNameHelp validationNameFeedback"
+                                   placeholder="e.g. MyNickname" required>
+                            <div id="validationNameHelp" class="form-text">The users display name.</div>
                             <div class="valid-feedback">{{ __("Looks good!") }}</div>
-                            <div class="invalid-feedback">{{ __("Please provide a valid name.") }}</div>
+                            <div id="validationNameFeedback" class="invalid-feedback">{{ __("Please provide a valid name.") }}</div>
```

Comment:
[INFO]

In a perfect world, the HTML attribute `id` is unique in the entire application - at least on the same HTML page.

When you include this blade multiple times (for each user), you will have the same ID multiple times.

A HTML validator would complain about a duplicate ID, but in general it isn't that bad as long as you don't apply any specific CSS, jQuery/JavaScript things to individual IDs differently.

This thread can be closed, if you're aware about the consequences. :)

Reply:
Thats right. I added to the modal edit blades their model `id` as an unique identifier for the html `id` attribut.